pom.xml

```diff
@@ -559,6 +559,14 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
Once the goal is to test against Java 20 only, we don't need this profile.

Could you remove this one?

Reply:
Sure! I created this profile to be able to override the project default settings which set the maven compiler plugin to use java 17, so I could use it on the ci to set the compiler settings to target java 20. 
But if the goal is to just check to compile with jdk 20, it isn't needed indeed. I'll remove it

```
<maven.compiler.source>17</maven.compiler.source>
<maven.compiler.target>17</maven.compiler.target>
```